geo.txt

```diff
@@ -56,3 +56,731 @@ Phys 91(11):957-962
+# Source Paper: Varga Z, Kolonits M, Hargittai M (2012) Comprehensive study of the structure of aluminum trihalides from
+electron diffraction and computation. Struct Chem 23 (3):879-893 
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Al Cl 2.073(4)
+
+# Bond lengths: re(M)  (angstroms)
+
+Al Cl 2.061(5)
+
+NAME = AlF3 
+# Point Group: D3h
+# Shape: Trigonal planar
+# Bond lengths: rg(a)     (angstroms)
+# Source Paper: a. Varga Z, Kolonits M, Hargittai M (2012) Comprehensive study of the structure of aluminum trihalides from
+electron diffraction and computation. Struct Chem 23 (3):879-893
+b. Hargittai M, Kolonits M, Tremmel J, Fourquet J-L, Ferey G (1990) The molecular geometry of iron trifluoride
+from electron diffraction and a reinvestigation of aluminum trifluoride. Struct Chem 1:75-78 
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Al F 1.632(3)
+
+# Bond lengths: re(M)(a,b)  (angstroms)
+
+Al F 1.621(4)
+
+NAME = ArH2
+# Point Group: C2v
+# Shape: Linear --- Linear
+# Bond lengths: r0     (angstroms)
+# Source Paper: McKellar ARW (2013) High resolution infrared spectra of H2-Xe and D2-Xe van der Waals complexes. Can J
+Phys 91(11):957-962
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Rcm(b) 3.957
+(b) Distance between Ar and the center-of-mass of H2.
+
+NAME = AsCl3 2009
+# Point Group: C3v
+# Shape: Trigonal pyramidal
+# Bond lengths: rh1     (angstroms)
+# Bond angles:  thetah1 (degrees)
+# Source Paper: Noble-Eddy R, Masters SL, Rankin DWH, Robertson HE, Guillemin JC (2010) Molecular structures of
+vinylarsine, vinyldichloroarsine and arsine studied by gas-phase electron diffraction and quantum chemical
+calculations. J Mol Struct 978 (1-3):26-34 
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+As Cl    2.164(4)
+Cl As Cl 98.9(3) 
+
+NAME = AsHO
+# Point Group: Cs
+# Shape: Bent
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Grimminger R, Clouthier DJ (2011) The electronic spectrum of the previously unknown HAsO transient
+molecule. J Chem Phys 135(18):184308/1-184308/8
+doi:10.1063/1.3658645
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+H As   1.576(3)
+As=O   1.6342(5)
+H As=O 101.5(4)
+
+NAME = AsH2
+# Point Group: C2v
+# Shape: Bent
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Grimminger RA, Clouthier DJ (2012) Toward an improved understanding of the AsH2 free radical: Laser
+spectroscopy, ab initio calculations, and normal coordinate analysis. J Chem Phys 137(22):224307/1-224307/12
+http://dx.doi.org/10.1063/1.4769778
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+As H    1.519(1)
+H  As H 90.75(9)
+
+# Bond lengths: rz     (angstroms)
+# Bond angles:  thetaz (degrees)
+
+As H    1.5322(8)
+H  As H 90.7(2)
+
+# Bond lengths: re     (angstroms)
+# Bond angles:  thetae (degrees)
+
+As H    1.5169(8)
+H  As H 90.7(2)
+
+NAME = AsP3 2010
+# Point Group: C3v
+# Shape: Tetrahedral
+# Bond lengths: rg     (angstroms)
```

Comment:
In addition to the rg values at T = 388 K, there's also r0 values that are given in the book, but seem to be accidentally neglected here.

Reply:
I also added the source for the other experiment.
```
NAME = AsP3
# Point Group: C3v (assumed)
# Shape: Tetrahedral
# Bond lengths: rg     (angstroms)
# Source Paper: a. Cossairt BM, Cummins CC, Head AR, Lichtenberger DL, Berger RJF, Hayes SA, Mitzel NW, Wu G (2010)
On the molecular and electronic structures of AsP3 and P4. J Am Chem Soc 132 (24):8459-8465 b. Daly AM, Cossairt BM, Southwood G, Carey SJ, Cummins CC, Kukolich SG (2012) Microwave spectrum of
arsenic triphosphide. J Mol Spectrosc 278:68-71
# Date first added: 2023-07-05 (by Sichao Xiong)

# Temperature = 388 Kelvin

As P 2.1949(28)
P  P 2.3041(12)

# Bond lengths: r0     (angstroms)

As P 2.311
P  P 2.201
```